mlrun/datastore/v3io.py

```diff
@@ -144,13 +145,15 @@ def _put(self, key, data, max_chunk_size: int = ONE_GB):
             data = memoryview(data)
         except TypeError:
             pass
+        requests_session = requests.Session()
```

Comment:
As mentioned above, we should probably reuse the session at the class level, but as the code is right now, the session should be closed, preferably like this:
```suggestion
       with requests.Session() as requests_session:
```